src/openforms/submissions/models/submission_value_variable.py

```diff
@@ -296,6 +296,13 @@ class SubmissionValueVariable(models.Model):
         blank=True,
         auto_now=True,
     )
+    is_initially_prefilled = models.BooleanField(
```

Comment:
Actually `SubmissionValueVariable` already had a source field, but this could be overwritten by user_input/logic (when we eventually sort out the source field to be set by these changes). Then for the prefill data check we wouldn't retrieve these variables.

So I added a new field to track the prefill